src/raft_baselines/scripts/raft_predict.py

```diff
@@ -83,6 +83,8 @@ def make_predictions(
     random_seed,
 ):
     classifier = classifier_cls(train_dataset, **classifier_kwargs, **extra_kwargs)
+    print(f"MODEL TYPE: {classifier.tokenizer.tokenizer.name_or_path}")
```

Comment:
Does this work for classifiers that aren't `TransformerCausalLMClassifier`s?

Reply:
Good catch, this was a debug statement and will be removed :)